dataprofiler/profilers/data_labeler_column_profile.py

```diff
@@ -233,6 +233,7 @@ def profile(self):
         Property for profile. Returns the profile of the column.
         """
         profile = {
+            "data_label": self.data_label,
```

Comment:
@JGSweets @AnhTruong I've made a small refactor to put the data_label in the profile instead of having it extracted from the compiler level. Was there a specific reason the data_label was left out of the profile? Is this refactor inappropriate?

Reply:
I think this is appropriate.

The update: `profile["data_label"] = col_profile.pop("data_label")` completes the refactor.